### PR TITLE
[CROSSDATA-279] [CORE] Create External Table Base

### PR DESCRIPTION
**Pull request by luismcl**

### CODE REVIEW

File: core/src/main/scala/org/apache/spark/sql/crossdata/execution/datasources/ddl.scala [hunk 1]
```diff
@@ -119,6 +119,21 @@ private[crossdata] case class CreateView(viewIdentifier: TableIdentifier, queryP
...
+                                userSpecifiedSchema: StructType,
+                                provider: String,
+                                options: Map[String, String]) extends LogicalPlan with RunnableCommand {
+
+
+  override def run(sqlContext: SQLContext): Seq[Row] = {
+
+    val resolved = ResolvedDataSource.lookupDataSource(provider).newInstance()
 
+    val tableManipulation = resolved.asInstanceOf[TableManipulation]
```
Comment (darroyocazorla) [inline core/src/main/scala/org/apache/spark/sql/crossdata/execution/datasources/ddl.scala]: The datasource may not be a TableManipulation, so we should define how to handle it. Options:
 -> Throw an exception 
->  Use sys.error like most spark runnable commands

**Comment by miguel0afd** [comment]
Once the change suggested by @darroyocazorla had been committed, :+1:

**Comment by pfcoperez** [comment]
+1

**Comment by darroyocazorla** [comment]
+1

**Pull request merged**